### PR TITLE
[s] Ports Sanic The Hedgehog as a race for April Fools

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
rscadd: Bay Medical has been ported in its entirety.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
:-1:

**Comment by lzimann** [comment]
can we ban goof until april 2nd so he stops

**Comment by TheNightingale** [comment]
I was hoping you'd actually ported Baymed there...

**Comment by lzimann** [comment]
\>actually wanting baymed in tg

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -763,3 +764,23 @@ SYNDICATE BLACK OPS
...
+
+/datum/species/sanichu
+	name = "Sanichu"
+	id = "sanichu"
+	speedmod = -1
+	roundstart = 1
+	siemens_coeff = 0
+	sexes = 0
+
+/datum/species/sanichu/spec_life(mob/living/carbon/human/H)
```
Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/species_types.dm]: not for sanics tho? also sanichus should be children of sanics.

**Comment by Dorsisdwarf** [comment]
PR descriptions must be an accurate description of the contents 


(u fuk)

**Comment by Iamgoofball** [comment]
hey dipshit reopen this it's for the april fools branch in case you couldn't fucking read

**Comment by stapler2025** [comment]
durr

**Comment by Bawhoppen** [comment]
before it gets reopened please use the [s] tag before some curious player clicks on it in ooc

**Comment by Dorsisdwarf** [comment]
There isn't that better.

**Comment by Bawhoppen** [comment]
also kor wanted all these to be closed until april fools day anyways though

**Comment by Iamgoofball** [comment]
that was before the branch was made

**Comment by Bawhoppen** [comment]
no

**Comment by Bawhoppen** [comment]
[13:48:23] <TheGhostOfWhibyl1> [-tg-station] KorPhaeron closed pull request #16286: [s] minor fix (April-fools-branch...elo) https://github.com/tgstation/-tg-station/pull/16286
[13:48:29] <TheGhostOfWhibyl1> [-tg-station] KorPhaeron closed pull request #16283: [s]No more stuns (April-fools-branch...topkek) https://github.com/tgstation/-tg-station/pull/16283
[13:48:35] <TheGhostOfWhibyl1> [-tg-station] KorPhaeron closed pull request #16285: Fixes a VV runtime (April-fools-branch...vvfix) https://github.com/tgstation/-tg-station/pull/16285
[13:48:36] <Bawhoppen> the fuck kor
[13:48:56] <TheGhostOfWhibyl1> [-tg-station] KorPhaeron closed pull request #16122: FINE YOU ASSHOLES I AM SICK AND TIRED OF ALL THIS REVERTS GOOFCHEM (master...sonichu) https://github.com/tgstation/-tg-station/pull/16122
[13:49:04] <+KorPhaeron> stop spoiling april fools
[13:49:08] <+KorPhaeron> by opening it a week+ in advance
[13:49:16] <Bawhoppen> it is 
[13:49:19] <Bawhoppen> in a hidden branch
[13:49:25] <Bawhoppen> why you think we did the [s]
[13:49:26] <%Antur> "hidden"
[13:49:34] <+KorPhaeron> "hiddeN" on the front page
[13:49:37] <+KorPhaeron> of our repo

**Comment by Bawhoppen** [comment]
do changelogs actually work in a seperate branch

**Comment by Niknakflak** [comment]
shit pr

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -763,3 +764,23 @@ SYNDICATE BLACK OPS
 	specflags = list(RADIMMUNE,VIRUSIMMUNE,NOBLOOD,PIERCEIMMUNE,EYECOLOR)
 	sexes = 0
 
+
+
+/datum/species/sanic
+	name = "Sanic"
+	id = "sanic"
+	roundstart = 1
+	speedmod = -1
```
Comment (Malkevin) [inline code/modules/mob/living/carbon/human/species_types.dm]: Make them really slow instead, it'll be funnier.

Comment (Bawhoppen) [inline code/modules/mob/living/carbon/human/species_types.dm]: make them get faster with stamina damage

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/species_types.dm]: Make it straight up be -INFINITY

File: config/game_options.txt [hunk 3]
```diff
@@ -255,31 +255,35 @@ DEFAULT_LAWS 1
...
+## roundstart fasts
 ##-------------------------------------------------------------------------------------------
-## Uncommenting races will allow them to be choosen at roundstart while join_with_muntant_race is on
+## dont uncomment any they dont gofast
 
-## Disabling humans is possible but a bit buggy. Some code relies on the notion that white people exist in space.
-ROUNDSTART_RACES human
+## gota gofast
+ROUNDSTART_RACES sanic
+ROUNDSTART_RACES sanichu
```
Comment (Core0verload) [inline config/game_options.txt]: Configs are not pulled from repo by tg server. Any changes made by you will be lost. Thus, the PR will break.

**Comment by Bawhoppen** [comment]
goof stop trying to hog all of the april fools to yourself, just make it a selectable race and call it good

**Comment by KorPhaeron** [comment]
Not finished in time. You're too slow

**Comment by PKPenguin321** [comment]
All around me are familiar faces

**Pull request closed without merging**